README.md

```diff
@@ -20,7 +20,10 @@ Implementation of mDoc security according to [ISO/IEC 18013-5](https://www.iso.o
 ## Session encryption
 Session encryption uses standard ephemeral key ECDH to establish session keys for authenticated symmetric encryption.
 The ``SessionEncryption`` struct implements session encryption (for the mDoc currently)
-It is initialized from a) the session establishment data received from the mdoc reader, b) the device engagement data generated from the mdoc and c) the handover data.
+It is initialized from 
+* the session establishment data received from the mdoc reader.
```

Comment:
@Ameenk149, it seems we should maintain an `ordered` list as initially decided by the author of the file and use `#` instead of `*`.

Reply:
if we want to do ordered then we might need `<ol>`

Reply:
```
1. foo
1. bar
1. buzz
``` 

works as well: 

1. foo
1. bar
1. buzz